docs/QUICK_START.md

```diff
@@ -0,0 +1,193 @@
+
+## 2. Assemble your Tx module if required and install antennas on both the receiver and Tx module.
+
+If you are using one of the MatekSys Tx module kits, print a JR bay case from the STL download available from [MatekSys](https://www.mateksys.com/?portfolio=mr900-30-tx#tab-id-1).
+
+If you will use the MatekSys HC-04 module to connect your ground station via Bluetooth, set all three [dip switches](https://MATEKSYS.md#tx-module-hc-04-bluetooth-notes) to the on position befoer assembling your Tx module.
+
+If you are using the [R9M Tx module](FRSKY_R9.md#r9m-tx-module):
+- set dip switch 1 to the On position to configure it for serial data
+- Install the [Arduino IDE](https://www.arduino.cc/en/software) and use it to build and flash [firmware](WIRELESS_BRIDGE.md) on your ESP32 module.  Be sure to select the R9M variant for your module when you edit the mlrs-wireless-bridge.ino.
+- Connect your [ESP32 wireless bridge](https://FRSKY_R9.md#esp32-wireless-bridge) module to the serial and power pins of your R9M.
+
+When choosing and connecting antennas be aware of the difference between SMA and RP-SMA.  It is, unfortunately, possible to attach an RP-SMA antenna to an SMA jack but there will be no electrical connection of the signal.
+
+## 3. Connect your receiver to your flight controller
+
+For the 1 watt MatekSys [mLRS receivers](https://www.mateksys.com/?portfolio=mr24-30#tab-id-2):
+- connect the receiver Rx1 to a FC UART Tx.
+- connect the receiver Tx1 to the corresponding FC UART Rx.
+- Connect the receiver V pin to a 5 volt output on your FC.
+- Connect the receiver G pin to a ground pad on your FC.
+- If you will use CRSF for RC channels (recommended), connect the receiver TX2 to a different UART Rx on your FC.
+
+For most ELRS receivers, the [connections](ELRS_RECEIVERS.md#connections) are similar to those above except CRSF is not available.
+
+Connect your ground station to your ArduPilot flight controller via USB and configure the serial port (x) you wired to your receiver serial port.  Don't forget to press "write parameters" if your ground station requires this step.
+- Set SERIALx\_PROTOCOL to "MAVLink2" (2).
+- Set SERIALx\_BAUD to "57600" (57) (default mLRS Rx Ser Baudrate).
+
+If you will use CRSF or SBUS for RC channels, use your ground station to configure the serial port (x) you wired to your receiver RC output.
+- Set SERIALx_PROTOCOL to "RCIN" (23).
+- Set RC_PROTOCOLS to 536.
+- Set RSSI_TYPE to 5.
+
+## 4. Install firmware if required
+
+If both your Rx and Tx modules came pre-installed with mLRS firmware as with the MatekSys mLRS modules, you can use the existing firmware unless you need a bug fix or feature which was added in a newer firmware version.
```

Comment:
Since flashing is so easy - I think it makes since to have folks update to the latest at the start.

Reply:
I'd like to take advantage of the pre-installed firmware to make getting started as quick and easy as possible.  How about if we suggest updating later after they are familiar with the system?

Reply:
Fair enough - probably also worth mentioning that parameters will persist when upgrading to further encourage.